### DISCUSSION TITLE
2.2.0-beta.1 Released!

### DISCUSSION THREAD

**Discussion by smolinari** [quasarframework/app-extension-apollo#150] [Announcements]
Thanks to @IlCallo and @yusufkandemir for their work!

Adds TypeScript detection - #148 
Bumps and externalizes deps-  #147 

<hr /><em>This discussion was created from the release <a href='https://github.com/quasarframework/app-extension-apollo/releases/tag/2.2.0-beta.1'>2.2.0-beta.1 Released!</a>.</em>

**Comment by WCN-llc** [comment]
Hi,
Is this enough to plug in the new version?:
"The extension will add a configuration file into src/apollo and a boot file. You'll need to manually register the latter into quasar.conf.js > boot."

Or should one be guided entirely by this?:
[https://dev.to/quasar/quasar-and-apollo-client-state-without-vuex-2iii](url)

Would like full documentation...

**Reply by smolinari** [reply]
@WCN-llc - The readme here is the full documentation for the usage of the app extension. That article is very old and was written when v1 was the up-to-date version. The concepts in the article still stand though. 

Scott

**Comment by WCN-llc** [comment]
> @WCN-llc - The readme here is the full documentation for the usage of the app extension. That article is very old and was written when v1 was the up-to-date version. The concepts in the article still stand though.
> 
> Scott

Thank you!
